lib/facter/winlogbeat_version.rb

```diff
@@ -0,0 +1,10 @@
+require 'facter'
+Facter.add('winlogbeat_version') do
+  confine :kernel => %w(Windows) # rubocop:disable Style/HashSyntax
+  if File.exist?('c:\Program Files\Winlogbeat\winlogbeat.exe')
```

Comment:
Unfortunately no. Facts do not have access to manifests or their variables...
Facts are loaded first then sent to be used in the manifests.